s/features/chat/components/xiome-chat/renderers/render-chat-post.ts

```diff
@@ -11,24 +13,28 @@ export function renderChatPost({
 		mute: () => void
 		remove: () => void
 	}) {
+	let postTime = formatDate(post.time)
 	return html`
 		<li data-post="${post.postId}">
 			<header>
+				<span>
 				<span class=nickname>${post.nickname}</span>
-				<span class=userid>${post.userId}</span>
-			</header>
-			<p class=content>${post.content}</p>
-			<footer>
-				<span class=time>${formatDate(post.time).full}</span>
+				<xio-id class="userid" title="${post.userId}" id="${post.userId}"></xio-id>
+				</span>
 				${isModerator
 					? html`
 						<span>
-							<xio-button @press=${mute}>mute</xio-button>
-							<xio-button @press=${remove}>delete</xio-button>
+							<xio-button style="border: none;" title="Mute" @press=${mute}>${muteIcon}</xio-button>
```

Comment:
- we shouldn't use `style` attributes in our html. this should be applied via css

Reply:
Whoops, that must've slipped through when i was setting up new pc